lib/services/sshd.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
```

Comment:
copyright date

Reply:
> > Again, the purpose of the test is to probe that the ssh server is accessible _from outside_ and _right after install_. Meaning, with the firewall up.
> 
> I think we are almost in the same page, just there was some misunderstanding :)
> 
> I thought we could reuse more from those tests and you found some issue that you didn't mentioned, but it is not the case, your explanation about 'from outside' confused me because both are not testing anything from outside, one is more simple that the other and does not require the firewall down, because does not try connection client/server in localhost.
> 
> My bad about the X11 fw, I thought that the original idea of the ticket was to check X11 forwarding, but it is not mentioned in the ticked explicitly, i found it in other bugs so I guess it is optional, I found interesting for example to check in `/etc/ssh/sshd_config` X11Forwarding Yes. Do you think that we could check anything else there?

I guess I could add something like "ssh root@localhost 'true'", which would return 0 if the connexion works.